crates/conformance-test-utils/src/hive/genesis.rs

```diff
@@ -265,9 +265,10 @@ mod tests {
     #[tokio::test]
     async fn test_madara_genesis() {
         // Given
-        let hive_genesis = HiveGenesisConfig::new().expect("Failed to read genesis.json");
-        let madara_loader = serde_json::from_str::<GenesisLoader>(std::include_str!("../madara/genesis.json")).unwrap();
-        let madara_genesis = Path::new("src/hive/madara_genesis.json");
+        let hive_genesis = HiveGenesisConfig::from_file("./src/test_data/hive_genesis.json").unwrap();
+        let madara_loader =
+            serde_json::from_str::<GenesisLoader>(std::include_str!("../test_data/madara_genesis.json")).unwrap();
+        let madara_genesis = Path::new("./src/test_data/combined_genesis.json");
```

Comment:
nit: `combined_genesis` instead of `madara_genesis`. I think we might have to change the parameter from `combined_genesis` than `madara_genesis` in `serialize_hive_to_madara_genesis_config` as it's more desriptive